samples/outlook/99-preview-apis/get-in-reply-to.yaml

```
@@ -0,0 +1,56 @@
+order: 7
+id: outlook-get-in-reply-to
+name: Get InReplyTo (Message Compose)
+description: Retrieves the InReplyTo property of the message being composed.
+host: OUTLOOK
+api_set:
+    Mailbox: preview
+script:
+    content: |
+        $("#get-in-reply-to").click(getInReplyTo);
+
+        function getInReplyTo() {
+          // This snippet returns the InReplyTo property of the message (PR_IN_REPLY_TO_ID).
+          // The API call works on a message being composed and is not supported on read items.
+          var inReplyTo = Office.context.mailbox.item.inReplyTo;
+          if (inReplyTo)
+          {
+            console.log("InReplyTo: " + inReplyTo);
+          }
+          else
+          {
+            console.log("No InReplyTo property available for this message");
+          }
+        }
+    language: typescript
+template:
+    content: |
+        <button id="run" class="ms-Button">
+            <span class="ms-Button-label">Run</span>
+        </button>
```

```suggestion
        <section class="ms-font-m">
	    <p class="ms-font-m">This sample shows how to get the ID of the message being replied to by the current message.</p>
	    <p><b>Required mode</b>: Message Compose</p>
        </section>

        <section class="samples ms-font-m">
	    <h3>Try it out</h3>
	    <button id="get-in-reply-to" class="ms-Button">
		<span class="ms-Button-label">Get the ID</span>
            </button>
        </section>
```
Suggesting to add an explanation to the task pane for consistency with other samples.